### PR TITLE
Change explorer.exe path

### LINKED ISSUE #2: [v2.2] Sandboxed program has crashed: explorer.exe
When you right-click any sandbox from the SandboxToys2 menu -> Explore > Sandboxed:

crash

Note: it's a clean standard sandbox.

### PR DESCRIPTION
**Pull request by isaak654**
See also https://github.com/sandboxie-plus/Sandboxie/blob/8beec95b4ef437de5b19c5f44dc8d60116b1959a/Sandboxie/apps/control/MyFrame.cpp#L1393-L1399

Fixes #2

### CODE REVIEW

File: SandboxToys2.ahk [hunk 1]
```diff
@@ -4468,27 +4468,27 @@ Return
 SExploreMenuHandler:
     box := getBoxFromMenu()
     if (GetKeyState("Control", "P"))
-        writeSandboxedShortcutFileToDesktop(start,"Explore sandbox " box " (Sandboxed)","","/box:" box " explorer.exe","Launches Explorer sandboxed in sandbox " box,"explorer.exe",1,1, box)
+        writeSandboxedShortcutFileToDesktop(start,"Explore sandbox " box " (Sandboxed)","","/box:" box " explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}","Launches Explorer sandboxed in sandbox " box,"explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",1,1, box)
```
Comment (isaak654) [inline SandboxToys2.ahk:4471]: I'm not sure whether `box,"explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",1,1, box)` change is required.

File: SandboxToys2.ahk [hunk 2]
```diff
@@ -4468,27 +4468,27 @@ Return
...
-        run, %start% /box:%box% explorer.exe, , UseErrorLevel
+        run, %start% /box:%box% explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}, , UseErrorLevel
 Return
 
 UExploreMenuHandler:
     box := getBoxFromMenu()
     path := sandboxes_array[box,"path"]
     if (GetKeyState("Control", "P"))
-        writeUnsandboxedShortcutFileToDesktop(WinDir "\explorer.exe","Explore sandbox " box " (Unsandboxed)",path,A_Quotes path A_Quotes,"Launches Explorer unsandboxed in sandbox " box,"explorer.exe",1,1)
+        writeUnsandboxedShortcutFileToDesktop(WinDir "\explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}","Explore sandbox " box " (Unsandboxed)",path,A_Quotes path A_Quotes,"Launches Explorer unsandboxed in sandbox " box,"explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",1,1)
```
Comment (isaak654) [inline SandboxToys2.ahk:4480]: I'm not sure whether `box,"explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",1,1)` change is required.

File: SandboxToys2.ahk [hunk 3]
```diff
@@ -4468,27 +4468,27 @@ Return
...
-        run, explorer.exe "%path%", , UseErrorLevel
+        run, explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D} "%path%", , UseErrorLevel
 Return
 
 URExploreMenuHandler:
     box := getBoxFromMenu()
     path := sandboxes_array[box,"path"]
     if (GetKeyState("Control", "P"))
-        writeUnsandboxedShortcutFileToDesktop(WinDir "\explorer.exe","Explore sandbox " box " (Unsandboxed, restricted)",path,"/root`,"A_Quotes path A_Quotes,"Launches Explorer unsandboxed and restricted to sandbox " box,"explorer.exe",1,1)
+        writeUnsandboxedShortcutFileToDesktop(WinDir "\explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}","Explore sandbox " box " (Unsandboxed, restricted)",path,"/root`,"A_Quotes path A_Quotes,"Launches Explorer unsandboxed and restricted to sandbox " box,"explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",1,1)
```
Comment (isaak654) [inline SandboxToys2.ahk:4489]: I'm not sure whether `box,"explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",1,1)` change is required.

**Review by blap — APPROVED** [review]
I wiil try it

**Comment by blap** [comment]
Updated.

**Pull request merged**